src/components/WelcomeProton/WelcomeProton.scss

```diff
@@ -0,0 +1,225 @@
+  color: #ffffff;
+  margin-top: 102px;
+  padding-right: 10%;
+  padding-left: 20%;
+  @include breakpoint($breakpoint-medium-landscape) {
+    padding-right: 5%;
+    padding-left: 6%;
+  }
+  @include breakpoint($breakpoint-small) {
+    margin-top: 222px;
+    padding-right: 5%;
+    padding-left: 10%;
+  }
+}
+.WelcomeProtonDesc {
+  color: var(--unnamed-color-ffffff);
+  text-align: left;
+  font: normal normal normal 19px/25px Arial;
+  letter-spacing: 0px;
+  color: #ffffff;
+  opacity: 1;
+  padding-right: 258px;
+  padding-left: 139px;
+  @include breakpoint($breakpoint-medium-landscape) {
+    padding-right: 5%;
+    padding-left: 6%;
+  }
+  @include breakpoint($breakpoint-small) {
+    top: 401px;
+    left: 40px;
+    padding: 0;
+    width: 286px;
+    height: 46px;
+    position: absolute;
+  }
+}
+.WelcomeProtonText {
+  color: var(--unnamed-color-ffffff);
+  text-align: left;
+  font: normal normal bold 19px/25px Arial;
+  letter-spacing: 0px;
+  color: #ffffff;
+  opacity: 1;
+  padding-right: 106px;
+  padding-left: 139px;
+  @include breakpoint($breakpoint-medium-landscape) {
+    padding-right: 5%;
+    padding-left: 6%;
+  }
+  @include breakpoint($breakpoint-small) {
+    top: 503px;
+    left: 6%;
+    position: absolute;
+  }
+}
+.ArrowLeft {
+  border: 2px solid var(--unnamed-color-ffffff);
+  border: 2px solid #ffffff;
+  opacity: 1;
+  width: 82px;
+  height: 0px;
+  top: 488px;
+  left: 139px;
+  position: absolute;
+  @include breakpoint($breakpoint-small) {
+    top: 717px;
+    left: 48px;
+    width: 0px;
+    height: 113px;
+    /* transform: matrix(0, 1, -1, 0, 0, 0); */
+    border: 2px solid var(--unnamed-color-ffffff);
+    border: 2px solid #ffffff;
+    opacity: 1;
+    position: absolute;
+  }
+}
+.arrow {
+  display: inline-block;
+  width: 15px;
+  height: 15px;
+  border-top: 3px solid #fff;
+  border-right: 3px solid #fff;
+  position: absolute;
+  top: 482px;
+  left: 208px;
+  @include breakpoint($breakpoint-small) {
+    display: inline-block;
+    border-top: 3px solid #fff;
+    border-right: 3px solid #fff;
+    position: absolute;
+    top: 816px;
+    left: 43px;
+    opacity: 1;
+    width: 15px;
+    height: 15px;
+  }
+}
+.arrow-right {
+  transform: rotate(45deg);
+  @include breakpoint($breakpoint-small) {
+    transform: rotate(135deg);
+  }
+}
+
+.IconFb {
+  top: 681px;
+  left: 139px;
+  width: 14px;
+  height: 27px;
+  background: var(--unnamed-color-657ee4) 0% 0% no-repeat padding-box;
+  opacity: 1;
+  position: absolute;
+  @include breakpoint($breakpoint-medium-landscape) {
+    left: 6%;
+  }
+  @include breakpoint($breakpoint-small) {
+    top: 40px;
+    left: 41px;
+    width: 12px;
+    height: 23px;
+    background: var(--unnamed-color-657ee4) 0% 0% no-repeat padding-box;
+    opacity: 1;
+    position: absolute;
+  }
+}
+.IconTw {
+  top: 684px;
+  left: 186px;
+  width: 27px;
+  height: 22px;
+  background: var(--unnamed-color-657ee4) 0% 0% no-repeat padding-box;
```

Comment:
Please use Sass variables, I don't understand the meaning of `--unamed-color-657ee4`, this is `$pm-blue`.